OSDOCS-4439: Adding info on mDNS in MicroShift (#53878)
openshift/openshift-docs: modules/microshift-mDNS.adoc

// Module included in the following assemblies:
//
// * microshift_networking/microshift-networking.adoc

:_content-type: CONCEPT
[id="microshift-mDNS_{context}"]

> == mDNS

Reviewer: If this is the first mention of mDNS, it shouldn't be abbreviated, per IBMSG guidance on abbreviations.  Multicast DNS (mDNS)

> == mDNS

Reviewer: ```suggestion
== The multicast DNS Protocol
```

> mDNS is a protocol used to allow name resolution and service discovery within a Local Area Network (LAN) using multicast exposed on the `5353/UDP` port.

Reviewer: ```suggestion
The multicast DNS protocol (mDNS) allows name resolution and service discovery within a Local Area Network (LAN) using multicast exposed on the `5353/UDP` port.
```

{product-title} includes an embedded mDNS server for deployment scenarios in which the authoritative DNS server cannot be reconfigured to point clients to services on {product-title}.

> This allows `.local` domains exposed by {product-title} to be discovered by other elements on the LAN.

Reviewer: "This allows `.local`..." seems a little orphaned conceptually :/ can you incorporate it more with the preceding paragraph so the "this" isn't questionable?
